### ISSUE TITLE
GUI QoL improvements

### ISSUE THREAD

**Issue by QarthO** [YufiriaMazenta/Craftorithm#50]
### Describe your idea | 描述你的想法

# Sort by result in ``/craft list``

The sorting method seems random, and is really hard to follow. It would be nice if the blocks were sorted by default so that same blocks are next to each other

For example, this is really confusing to look at, since they are out of order. Having the same blocks next to each other will be helpful.
![image](https://github.com/YufiriaMazenta/Craftorithm/assets/10179096/8c47762c-f985-4e6a-9ca4-2e6c5593c0a4)

# Useful info in lore
There should be helpful info in the lore of each recipe. Having to click through each item is cumbersome.

# Edit existing recipe without deleting
I'd like to be able to modify an existing crafting recipe without having to delete it. Currently you can only edit the sort id, and the unlock recipe.

*Note: What even is sort id for? And what does unlock recipe do?

**Comment by YufiriaMazenta** [comment]
Sort_id is used for sorting. The craft list is sorted according to sort_id from small to large.
Unlock controls whether it is unlocked for players in the vanilla recipe book

**Comment by YufiriaMazenta** [comment]
The recipe editor gui will be optimized in version 2.0

**Comment by YufiriaMazenta** [comment]
Also, what kind of useful info do you mean? Can you explain it in detail?

**Comment by ArubikU** [comment]
viewing qarth0 dont response i think a good implement to lore can be a crafting preview using symbols like

A B A
B A B -> C
A B A

A = DIRT
B = BEDROCK
C = BEDROCK

**Comment by YufiriaMazenta** [comment]
The new GUI has been released – it supports custom menus and includes a recipe list menu that can be sorted by name or creation time.
